### ISSUE TITLE
Incorrect referencing of types decleration file

### ISSUE THREAD

**Issue by saddiqs1** [akiver/csgo-sharecode#75]
I get the following error when attempting to import the library using `import csgoSharecode from 'csgo-sharecode'`. This error only appeared for me in the recent release:

```
Could not find a declaration file for module 'csgo-sharecode'. '<path-to-project>/node_modules/csgo-sharecode/dist/esm/index.js' implicitly has an 'any' type.
  There are types at '<path-to-project>/node_modules/csgo-sharecode/dist/index.d.ts', but this result could not be resolved when respecting package.json "exports". The 'csgo-sharecode' library may need to update its package.json or typings.ts(7016)
  ```

**Comment by akiver** [comment]
Could you please provide a minimal, reproducible project example?
I can't reproduce the issue on a fresh Nodejs project.

**Comment by saddiqs1** [comment]
https://github.com/saddiqs1/csgo-sharecode-issue

Interestingly the code does actually work, however in vscode it displays the error in the original post, as well as losing all intellisense/typing suggestions

**Comment by akiver** [comment]
Thank you for the example.
Could you try with the new version 3.1.1 please? 
I tried locally and it seems to fix the issue.

**Comment by claabs** [comment]
Hi, this fixes the issue for me, however 3.1.1 is not published to npm. Can you please publish it?

**Comment by akiver** [comment]
> Hi, this fixes the issue for me, however 3.1.1 is not published to npm. Can you please publish it?

😓 sorry, it's on npm now.
